### PR TITLE
Test/test withdraw

### PR DESCRIPTION
**Pull request by casweeney**
## Changes description
- Wrote test for `withdraw()` function
- Made minor fix in the `withdraw()` function

### CODE REVIEW

File: contracts/.tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+scarb 2.6.5
```
Comment (EmmanuelAR) [inline contracts/.tool-versions:1]: Why did u add this @casweeney ?

File: contracts/tests/test_fund.cairo [hunk 2]
```diff
@@ -309,49 +309,75 @@ fn test_withdraw_with_non_closed_state() {
...
+    let owner_balance_before = token_dispatcher.balance_of(OWNER());
+    let fund_balance_before = token_dispatcher.balance_of(contract_address);
+
+    // withdraw
+    dispatcher.withdraw();
+
+    let owner_balance_after = token_dispatcher.balance_of(OWNER());
+    let fund_balance_after = token_dispatcher.balance_of(contract_address);
+
+    assert!(
```
Comment (EmmanuelAR) [inline contracts/tests/test_fund.cairo:368]: please use assert( like commented code please, avoid to use assert!(

**Review by EmmanuelAR — CHANGES_REQUESTED** [review]
thks for the pr please some change!

File: contracts/.tool-versions (hunk 1 above)
Comment (casweeney) [reply contracts/.tool-versions:1]: > Why did u add this @casweeney ?

I think add it manually. I followed the Readme instructions here: https://github.com/web3wagers/gostarkme/tree/dev/contracts#readme that required that I install and use `scarb 2.6.5` I guess it was added automatically.

File: contracts/tests/test_fund.cairo (hunk 2 above)
Comment (casweeney) [reply contracts/tests/test_fund.cairo:368]: > please use assert( like commented code please, avoid to use assert!(

Alright, let me fix that.

**Review by EmmanuelAR — APPROVED** [review]

**Pull request merged**